sdk/core/Azure.Core/api/Azure.Core.net461.cs

```diff
@@ -1145,12 +1144,17 @@ public static partial class ModelSerializer
     }
     public partial class ModelSerializerOptions
     {
-        public ModelSerializerOptions() { }
-        public bool IgnoreAdditionalProperties { get { throw null; } set { } }
-        public bool IgnoreReadOnlyProperties { get { throw null; } set { } }
+        public ModelSerializerOptions(string format = "Data") { }
+        public string FormatType { get { throw null; } }
         public string? NameHint { get { throw null; } set { } }
         public bool PrettyPrint { get { throw null; } set { } }
         public System.Collections.Generic.Dictionary<System.Type, Azure.Core.Serialization.ObjectSerializer> Serializers { get { throw null; } }
+        [System.FlagsAttribute]
+        public enum Format
```

Comment:
This should be implemented as an [extensible enum](https://azure.github.io/azure-sdk/dotnet_implementation.html#dotnet-enums) with values of Wire = "W" and Data = "D"